docs_src/microservices/configuration/Ch-Configuration.md

```diff
@@ -208,26 +220,22 @@ File = ''
 
 ## Configuration Provider
 
-You can supply and manage configuration in a centralized manner by utilizing the `-cp/--configProvider=<url>` flag when starting a service. If the flag is provided and pointed to an application such as HashiCorp's [Consul](https://www.consul.io/), the service will bootstrap its configuration into Consul if it doesn't exist. If configuration does already exist, it will load the content from the given location applying any environment variables overrides of which the service is aware. Integration with the configuration provider is handled through the
+You can supply and manage configuration in a centralized manner by utilizing the `-cp/--configProvider=<url>` flag when starting a service. If the flag is provided and pointed to an application such as [HashiCorp's Consul](https://www.consul.io/), the service will bootstrap its configuration into Consul if it doesn't exist. If configuration does already exist, it will load the content from the given location applying any environment variables overrides of which the service is aware. Integration with the configuration provider is handled through the
 [go-mod-configuration](https://github.com/edgexfoundry/go-mod-configuration) module referenced by all services.
 
 ## Registry Provider
 
-The registry refers to any platform you may use for service discovery. For the EdgeX Foundry reference implementation, the default provider for this responsibility is Consul. Integration with the registry is handled through the
-[go-mod-registry](https://github.com/edgexfoundry/go-mod-registry)
-module referenced by all services.
-
-![image](./EdgeX_RegistryHighlighted.png)
+The registry refers to any platform you may use for service discovery. For the EdgeX Foundry reference implementation, the default provider for this responsibility is Consul. Integration with the registry is handled through the [go-mod-registry](https://github.com/edgexfoundry/go-mod-registry) module referenced by all services.
 
 ### Introduction to Registry
 
-The objective of the registry is to enable microservices to find and to
-communicate with each other. When each microservice starts up, it
+The objective of the registry is to enable micro services to find and to
+communicate with each other. When each micro service starts up, it
 registers itself with the registry, and the registry continues checking
 its availability periodically via a specified health check endpoint.
-When one microservice needs to connect to another one, it connects to
+When one micro service needs to connect to another one, it connects to
```

Comment:
This isn't correct.
I Geneva it currently uses `Client Monitoring`, which keeps the service endpoint update with periodic calls to the Registry. No calls to registry each time we make a call to another service. In Hanoi we have removed Client Monitoring and depend on configuration rather that the registry.

Reply:
will remove